server/aws-lsp-yaml-json/src/language-service/yamlLanguageService.ts

```diff
@@ -29,7 +29,7 @@ export class YamlLanguageService implements AwsLanguageService {
             workspaceContext,
         })
 
-        this.yamlService.configure({ schemas: [{ fileMatch: ['*.yml'], uri: this.props.defaultSchemaUri }] })
+        this.yamlService.configure({ schemas: [{ fileMatch: ['*'], uri: this.props.defaultSchemaUri }] })
```

Comment:
Same question for extension, what's the intention for allowing any file extension?

Reply:
discussed this comment in slack, this change is ok